helpers/UuidPrimaryKeyTrait.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
```

Comment:
Just a question about all `PHP` files, should be there empty lines before and after license block?

Reply:
Not totally sure. In PSR-12, not yet approved, I can see this: https://github.com/php-fig/fig-standards/blob/master/proposed/extended-coding-style-guide.md#3-declare-statements-namespace-and-import-statements
But I'm not sure it means we should have blank lines before license block. In fact, most of the files in find in tao code do not have a blank line before license block, so I'll remove it here. Thanks for seeing this, @emgolubev!